app/Console/Commands/UserCreate.php

```diff
@@ -18,14 +19,19 @@ class UserCreate extends Command
      *
      * @var string
      */
-    protected $signature = 'user:create {name} {email} {password} {language} {repair_network_id}';
+    protected $signature = 'user:create {name} {email} {password} {language} {repair_network_id} 
+                           {--role=4 : Role ID (1=ROOT, 2=ADMINISTRATOR, 3=HOST, 4=RESTARTER, 5=GUEST, 6=NETWORK_COORDINATOR)}
+                           {--consent-past-data= : Consent date for past data in YYYY-MM-DD format, defaults to today}
```

Comment:
Is this comment correct? I don't think it defaults to today unless `--auto-consent` is set, right?

(Same with the other args)

Reply:
Addressed in [c4a6899](https://github.com/iFixit/restarters/pull/15/commits/c4a68992bb961b27f2857873e7d91ea41389338d)